crates/rooch-types/src/transaction/authenticator.rs

```diff
@@ -34,8 +34,8 @@ pub struct RoochAuthenticator {
 }
 
 impl BuiltinAuthenticator for RoochAuthenticator {
-    fn coin_id(&self) -> CoinID {
-        CoinID::Rooch
+    fn scheme(&self) -> u64 {
+        BuiltinScheme::Ed25519.flag().into()
     }
```

Comment:
We can rename the builtin scheme to builtin auth validator.

Reply:
Renamed BuiltinAuthValidator.